source/includes/_datasets.md

```diff
@@ -105,12 +104,52 @@ This endpoint lists all the datasets that can be edited by this user.
 Parameter | Default | Description
 --------- | ------- | -----------
 `where` <br> *string* | None | Filter expression used to restrict returned datasets ([ODSQL documentation](https://docs.opendatasoft.com/api/explore/v2.html#where-clause))
-`start` <br> *string* | 0 | Index of the first item to return
+`page` <br> *string* | 1 | Number of the page you want to retrieve. Pages numbers vary when changing the number of rows to retrieve.
 `rows` <br> *string* | 10 | Number of items to return. Max value: 100
 `sort` <br> *string* | None | Field on which to sort the results list
 `include_app_metas` <br> *string* | false | Explicitely request application metadata for each datasets
 `timezone` <br> *string* | UTC | Timezone applied on datetime fields in query and response
 
 ### Returns
 
-Returns a list of dataset objects.
\ No newline at end of file
+Returns a list of dataset objects.
+
+
+## Retrieve information about one dataset
```

Comment:
In the explore documentation, we use "Looking up a dataset", maybe we can use the same vocabulary in both?

Reply:
I agree we should be using the same vocabulary, but in this case I'd go for retrieve instead of look up. If I'm not mistaken, "look up" means "search" and retrieve is closer to "get".

Reply:
agreed, I would prefer we change the wording in the explore documentation